CONTRIBUTING.md

```diff
@@ -139,3 +138,30 @@ GitHub Action handles bumping the version of Gosling.js. The pattern looks like
 yarn version --patch
 git push origin master --tags
 ```
+
+# Internal Explanations  
+## How does a Gosling schema get turned into a HiGlass schema?
```

Comment:
```suggestion
## How does a Gosling spec get turned into a HiGlass spec?
```

I have been using `spec` and `schema` differently, spec as the JSON object and the schema as the grammar (TypeScript types). Let's use that convention.